src/TestSuite/TestConnectionManager.php

```diff
@@ -0,0 +1,94 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ * @since         4.3.0
+ * @license       https://opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Cake\TestSuite;
+
+use Cake\Datasource\ConnectionManager;
+
+/**
+ * This class provides utility for the management of the test connections.
+ */
+final class TestConnectionManager
+{
+    /**
+     * @var bool
+     */
+    public static $aliasConnectionIsLoaded = false;
+
+    /**
+     * @return void
+     */
+    public static function aliasConnections()
+    {
+        if (!self::$aliasConnectionIsLoaded) {
+            (new self())->_aliasConnections();
+            self::$aliasConnectionIsLoaded = true;
+        }
+    }
+
+    /**
+     * Unset the phinx migration tables from an array of tables.
+     *
+     * @param  string[] $tables Array of strings with table names.
+     * @return array
+     */
+    public static function unsetMigrationTables(array $tables): array
```

Comment:
Couldn't this go on `SchemaCleaner` as well? The other methods in this class are related to connection management. This method feels out of place.

Reply:
Agreed, done.

Reply:
I've cleaned up a bit that part of the code, which also convinced me that there was way too much staticity there.
Let me know if this works for you!